src/devtools/store.js

```diff
@@ -655,6 +702,10 @@ export default class Store extends EventEmitter {
           i = i + 3;
           break;
         default:
+          // ----------
+          // TODO: I get here with 0 sometimes after toggling suspense :-(
+          // ----------
```

Comment:
"with 0" meaning...  the next operation in the array? Yikes. That's concerning.

Reply:
Could this be because you didn't actually add the `TREE_OPERATION_CLEAR_CHILDREN` constant? so when you push the undefined value, it's just defaulting to 0?

Reply:
(in which case I kind of wonder how the above `TREE_OPERATION_CLEAR_CHILDREN` case would ever be hit...)